contracts/nois-gateway/src/contract.rs

```diff
@@ -278,40 +424,76 @@ fn execute_set_config(
         Some(dr) => Some(deps.api.addr_validate(&dr)?),
         None => config.drand,
     };
-    let price = match price {
-        Some(pr) => pr,
-        None => config.price,
+    let price = price.unwrap_or(config.price);
+    let payment_code_id = match payment_code_id {
```

Comment:
Do you think here it makes sense to make update payment_code_id more sophisticated in such a way that it handles the migration of the existing payment contracts?
I'm worried that such update power would create drifts in our payment contracts. Maybe have whether 
- a fully automatic update that updates the whole fleet (future and existing) 
- or do the whole migration of all payment contracts and gateway contract code id manually.

Reply:
Right. Let's remove it from here. We need something better later on.

Reply:
[6c47530](https://github.com/noislabs/nois-contracts/pull/196/commits/6c47530df04acc6eea90ecdf06912066fd6ec31c)